Lesson31/demo-app/Lesson31-merve2/src/shared-components/typography/BodyM/styles.css

```diff
@@ -0,0 +1,3 @@
+.body-m {
+    font-size: 55px;
```

Comment:
You can find details of the BodyM component here: https://www.figma.com/design/TktY1f8F29rlne47uLzqR2/frontend-quiz-app?node-id=260-943&node-type=frame&m=dev

It should use Rubik font, and have font-size 12px.